src/main/kotlin/io/github/rybalkinsd/kohttp/ext/ResponseExt.kt

```diff
@@ -76,3 +78,14 @@ data class EagerResponse(
  * @author sergey
  */
 data class Header(val name: String, val value: String)
+
+internal val stringMapper = ObjectMapper()
+fun String?.asJson(): JsonNode = if (isNullOrBlank()) stringMapper.readTree("{}") else stringMapper.readTree(this)
+
+internal fun Response.asJson() = with(body()?.string()) {
+    asJson()
+}
+
+internal fun Response.asString() = body()?.string()
```

Comment:
making these functions as internal it will be not possible to invoke them from client projects

Reply:
I did try with multiline strings initially, but they don't indent well. 
Here's to add a little more context.

https://github.com/rybalkinsd/kohttp/pull/84/commits/ea2e6e7372d5a0bab472ca3a221a97c342678f6b#diff-3d2eecefb292c407d3ecd7e81486e1a9L84